### ISSUE TITLE
ZOOP changing nets

### ISSUE THREAD

**Issue by joneslabND** [MFEh2o/db#3]
New net in 2017 may have had a much higher efficiency of capture????

**Comment by Randinotte** [comment]
Randi and Amaryllis grabbed test samples during Fall trip in 2019. Haven't counted those samples yet (covid). After count, could compare filter efficiency and adjust zoop data post-2017.

**Comment by kaijagahm** [comment]
@Randinotte just to double check, these still haven't been counted, have they?

**Comment by Randinotte** [comment]
@kaijagahm no, they keep getting kicked down the list with the rest of the non-Long zoops (from monitoring lakes, project 15). Maybe if I have time this summer I can get to those zoops.

**Comment by kaijagahm** [comment]
Ok, no worries! I'll mark this one as back burner for now.
